optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/uni/UniConstraintStream.java

```diff
@@ -197,8 +197,8 @@
             UniConstraintCollector<A, ResultContainer_, Result_> collector);
 
     /**
-     * Convert the {@link UniConstraintStream} to a different {@link UniConstraintStream}, containing of the set of
-     * tuples resulting from applying the group key mapping function on all tuples of the original stream.
+     * Convert the {@link UniConstraintStream} to a different {@link UniConstraintStream}, containing the set of tuples
+     * resulting from applying the group key mapping function on all tuples of the original stream.
      * Neither tuple of the new stream will {@link Objects#equals(Object, Object)} any other.
```

Comment:
I am not sure if these javadocs can't be made more "simple & clear". Also notice the big out-of-sync between the javadoc of groupBy(Function) and groupBy(Function, Collector)

Reply:
We should improve these (if possible), before copy-pasting it into the other streams cardinalities.